README.md

```diff
@@ -54,6 +54,30 @@ oc create secret generic apicast-configuration-url-secret \
 oc new-app -f https://raw.githubusercontent.com/3scale/apicast/master/openshift/apicast-template.yml
 ```
 
+### OKD (Openshift-origin 3.11)
```

Comment:
What does `OKD` mean? Perhaps this can be expanded, and then use the abbreviated form in the text.

Reply:
https://www.okd.io

I don't see where they explain what it stands for.

Reply:
In this case, the heading should be reworded. Suggestions: 

- Configuring 3scale APIcast with OKD
- Using OKD to configure 3scale APIcast

Then, the first sentence should define OKD: 
_OKD is a distribution of Kubernetes optimized for continuous application development and multi-tenant deployment._